config/settings/common.py

```diff
@@ -236,6 +238,8 @@
 ACCOUNT_EMAIL_REQUIRED = True
 ACCOUNT_EMAIL_VERIFICATION = 'mandatory'
 
+ACCOUNT_DEFAULT_HTTP_PROTOCOL = env.str("DJANGO_ACCOUNT_DEFAULT_HTTP_PROTOCOL", default="http")
```

Comment:
IMHO https should always be enforced by default settings in the modern world and anything less should be configured by the user. If someone really is going to deploy this app without https, make it difficult for them ;)